database/sqlite/sqlite_aclk_alert.c

```diff
@@ -125,7 +125,7 @@ void aclk_push_alert_event(struct aclk_database_worker_config *wc, struct aclk_d
     int rc;
 
     if (unlikely(!wc->alert_updates)) {
-        debug(D_ACLK_SYNC,"Ignoring alert push event, updates have been turned off for node %s", wc->node_id);
+        log_access("AC [%s (%s)]: Ignoring alert push event, updates have been turned off for this node.", wc->node_id, wc->host->hostname);
```

Comment:
It will be safer to check for `wc->host` being NULL

Reply:
Ok, thanks, added to all occurrences.